tests/test_vrs2.py

```diff
@@ -166,6 +166,24 @@ def test_vr():
                 "sequence": "T"
             }
         })
+    with pytest.raises(ValidationError):
+        models.Allele(**{
+            "type": "Allele",
+            "location": {
+                "type": "SequenceLocation",
+                "sequenceReference": {
+                    "type": "SequenceReference",
+                    "refgetAccession": "ga4gh:SQ.KEO-4XBcm1cxeo_DIQ8_ofqGUkp4iZhI"
+                },
+                "start": 128325834,
+                "end": 128325835
+            },
+            "state": {
+                "type": "LiteralSequenceExpression",
+                "sequence": "T"
+            },
+            "digest": "ga4gh:734G5mtNwe40do8F6GKuqQP4QxyjBqVp"
```

Comment:
I thought the plan was for the digest to be only the 'digest' without the "ga4gh:" prefix? @ahwagner  would have the final word on this.

Reply:
I'll make this section a little clearer

Reply:
@larrybabb is correct. The `digest` field is for the digest only. The `id` field _may_ contain a ga4gh-prefixed identifier per the VRS 2.0 spec, and VRS-Python _should_ do so.

Reply:
not enjoying these mixed signals. (@ahwagner this is in a `with pytest.raises` block)

Reply:
I think just update the line to: 

```python
"digest": "ga4gh:734G5mtNwe40do8F6GKuqQP4QxyjBqVp"  # this will fail because digest field should NOT include prefix
```

Reply:
Hey. Sorry, I didn't look at context, only at @larrybabb's comment. I think we're all on the same page: `ga4gh:` in the `digest` field BAD, testing that we catch those errors GOOD.